app/services/summary_service.rb

```diff
@@ -85,6 +101,10 @@ def self.sort_fields_for_new_arrivals
     [:last_name, :prison_arrival_date, :earliest_release_date]
   end
 
+  def self.sort_fields_for_handovers
```

Comment:
These `sorting` behaviours being in the `summary_service` as well as the `summary_controller` smells to me.

(I appreciate we're following an existing pattern, so I'm not saying change it now!)

Reply:
Sorting and pagination on the whole feels quite bespoke. I think we both assumed that a gem was being used to achieve this – but it turns out that's not the case.

I also noticed that the pagination/sort functionality in the view layer are disconnected from the controller layer – for example, the sorting helper [assumes `:last_name` is the default](https://github.com/ministryofjustice/offender-management-allocation-manager/blob/POM-566/app/helpers/sort_helper.rb#L8) sort column, despite [the controller having other opinions](https://github.com/ministryofjustice/offender-management-allocation-manager/blob/POM-566/app/controllers/summary_controller.rb#L90-L98).

Definitely something to look at when we can!